indico/htdocs/sass/partials/_jquery-indico-locationpicker.scss

```diff
@@ -0,0 +1,157 @@
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+.i-location-field {
+    max-width: 400px;
+
+    .wrapper {
+        float: left;
+        display: inline-block;
+
+        .typeahead-container,
+        .typeahead-field {
+            width: inherit;
+            display: inline-block;
+            position: relative;
+            z-index: 1030 !important;
+
+            .typeahead-results {
+                position: absolute;
+                left: 0;
+
+                .typeahead-list {
+                    margin: 0;
+                    padding-bottom: 0;
+                    overflow: auto;
+                    max-height: 30em;
+                    width: 20em;
+                    position: relative;
+                    border: none;
+                    border-radius: 0;
+                    -webkit-box-shadow: none;
+                    -moz-box-shadow: none;
+                    box-shadow: none;
+
+                    > li {
+                        background: white;
+                    }
+
+                    &.empty {
+                        li:first-child {
+                            display: none;
+                        }
+                    }
+                }
+
+                .just-use-option {
+                    display: none;
+                    cursor: pointer;
+                    background-color: $light-gray;
+                    text-align: center;
+                    padding: 6px 20px;
+
+                    &:hover {
+                        background-color: #ebebeb; // same as in typeahead
+                    }
+                }
+            }
+
+            &.backdrop {
+                .i-location-input-field {
+                    background-color: white;
+                    z-index: 1050 !important;
+                }
+
+                .just-use-option {
+                    display: block;
+                    border-top: 1px solid rgba(0, 0, 0, 0.15);
+                }
+
+                z-index: 1060 !important;
+            }
+        }
+
+        input.i-location-input-field {
+            width: 11.5em;
+            background-color: inherit;
+            padding-right: 2em !important;
+
+            &.disabled {
+                @extend %tilted-strips-bg;
+            }
+
+            &.bring-to-front {
+                background: white;
+            }
+        }
+
+        .toggle-results-list {
+            position: absolute;
+            padding: 0.5em 0.6em;
+            top: 0;
+            right: 0;
+            line-height: 1.5;
+            cursor: pointer;
+        }
+
+        > textarea {
+            @extend %form-field;
+            width: 100%;
+            margin-top: 0.4em;
+
+            &.disabled {
+                @extend %tilted-strips-bg;
+            }
+        }
+    }
+}
+
+.dropback-wrapper {
```

Comment:
backdrop, Allah dammit. Keyword used in the [Typeahead](http://www.runningcoder.org/jquerytypeahead/documentation/) library